src/App.tsx

```diff
@@ -181,6 +189,14 @@ const AppRoutes = ({ base }: AppProps) => {
                     />
                     <Route path="user/:username" element={<ArenaUserPage />} />
                   </Route>
+                  <Route path="admin">
+                    <Route index element={<ArenaAdminPage />} />
+                    <Route path={'users'} element={<ArenaUserListPage />} />
+                    <Route path={'flags'}>
```

Comment:
Eg liker denne måten å skrive slikt på.
```suggestion
                    <Route path="flags">
```

Reply:
Fikser 👍